src/__tests__/Parser.test.ts

```diff
@@ -155,13 +155,15 @@ test('parse throws an error if there are unparsed tokens', () => {
   }).toThrow(/.*unparsed tokens.*/)
 })
 
-test('is addition also valid multiplaction?', () => {
-  let parser = new Parser(scan('5 + 4'))
-  expect(() => {
-    parser.multiplication()
-  }).toThrow(/.*Addition is not valid multiplication.*/)
-})
-
-test('is multiplication also valid addition?', () => {
-  let parser = new Parser(scan('5 * 4'))
-})
+// Verify that the parser is not at the end
```

Comment:
why remove these tests?

Reply:
These are still in-progress 😅

Reply:
Got that fixed and checked to see if the parser was reaching the end of the input